packages/mdc-ripple/_mixins.scss

```diff
@@ -241,6 +243,142 @@
   }
 }
 
+//
+// New mixin APIs
+//
+
+@mixin mdc-ripple-surface() {
+  --mdc-ripple-fg-size: 0;
+  --mdc-ripple-left: 0;
+  --mdc-ripple-top: 0;
+  --mdc-ripple-fg-scale: 1;
+  --mdc-ripple-fg-translate-end: 0;
+  --mdc-ripple-fg-translate-start: 0;
+
+  will-change: transform, opacity;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+
+  &::before,
+  &::after {
+    position: absolute;
+    transition: opacity 250ms linear;
+    border-radius: 50%;
+    pointer-events: none;
+    content: "";
+  }
+}
+
+@mixin mdc-ripple-color_($color, $opacity) {
+  // stylelint-disable at-rule-empty-line-before, block-closing-brace-newline-after
+  @if type-of($color) == "color" {
+    background-color: rgba($color, $opacity);
+  } @else {
+    // Assume $color is a theme property name
+    $theme-value: map-get($mdc-theme-property-values, $color);
+    $css-var: var(--mdc-theme-#{$color}, $theme-value);
+
+    background-color: rgba($theme-value, $opacity);
+
+    // See: https://drafts.csswg.org/css-color/#modifying-colors
+    // While this is currently unsupported as of now, it will begin to work by default as browsers
+    // begin to implement the CSS 4 color spec.
+    @supports (background-color: color(green a(10%))) {
```

Comment:
mmmm...this might be outside the scope of this PR...but could we just remove the @supports? It's odd we use this technology before it is supported across all our supported browsers

Reply:
I definitely had thoughts of eliminating this, but I also thought it might make sense for me to evaluate this later with work on states, because I'm wondering if there's even a convincing reason for us to be using RGBA colors rather than relying more directly on the opacity property, which would also make this completely moot.